app/views/jst/conferences/concludedConference.handlebars

```diff
@@ -60,25 +60,55 @@
     <ul>
       {{#each recordings}}
         <li>
-          <div class="ig-row">
-            <div class="ig-row__layout">
-              <div class="ig-info">
-                <div class="ig-details">
-                  <a href="{{playback_url}}" target="_blank" class="ig-title">
-                    {{title}}
```

Comment:
we don't want to show the title anyone where a BBB conference?

Reply:
Well, the title has actually never been shown. I didn't go further because the recording title is always going to be the conference name anyways and the icon is not very explicit on what it is
![image](https://user-images.githubusercontent.com/578359/33093920-5b31f60c-cecc-11e7-9a0f-d93e0906aac8.png)

Reply:
There was a problem in the model. I added title based on recording.name

Reply:
I see, yeah, it looks like the title was coming through blank.

It'd be great to not have to have a big if/else on vendor with lots of duplicated code, but instead just add the code for the delete button if it's available for that recording.

Reply:
Agreed!. That was my main concern one year ago when we started with the first PR. I don't know if you remember, but there was this massive PR with about 10 new features :laughing: 

I honestly don't remember to whom (from Canvas) we were speaking with at that time, but we came to the conclusion that we were going to need to do a separation of code as BBB is offering functionality that no other is. The if was the only thing we had handy as it was the less complicated.

But yeah, we can do this kind of simplified implementation for the delete PR. Although sooner o later we are going to need to separate the code. I still have a long list of features (and the code somehow ready), but the agreement with Matt Goodwin is that we will do small PRs one at the time :smile: